core/mo-services-impl/nmf-software-management-impl/src/main/java/esa/mo/sm/impl/provider/AppsLauncherManager.java

```diff
@@ -119,6 +125,11 @@ public AppsLauncherManager(COMServicesProvider comServices)
       // With Archive...
     }
 
+    if(osValidator.isUnix()){
+        ShellCommander shell = new ShellCommander();
+        String out = shell.runCommandAndGetOutputMessageAndError("sudo --help");
+        sudoAvailable = !out.contains("command not found");
```

Comment:
Searching for `command not found` is not very portable. Better check if the return code of `sudo --help` is 0 or otherwise.

See example from a shell which would fail the above check.
```
root@sepp:~# sdo --help
-sh: sdo: not found
root@sepp:~# echo $?
127
```
```
root@sepp:~# sudo --help
sudo - execute a command as another user
[...]
root@sepp:~# echo $?
0
```

Reply:
Agreed